tests/mgbench/workloads/high_write_set_property.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+from workloads.base import Workload
+
+
+class HighWriteSetProperty(Workload):
+    NAME = "high_write_set_property"
+    CARDINALITY = 100000
+
+    def indexes_generator(self):
+        indexes = []
+        if "neo4j" in self.benchmark_context.vendor_name:
```

Comment:
There is no need to add Neo4j stuff for things that will not run on Neo4j. It is ok to skip that and return the Memgraph stuff.